cmd/internal/find/primary/meta.go

```diff
@@ -67,15 +67,29 @@ var Meta = Parser.add(&Primary{
 })
 
 const metaDetailedDescription = `
-The meta primary constructs a predicate on the entry's meta attribute.
-It is a specialized filter, so you should only use it if you need to
+The meta primary constructs a predicate on the entry's metadata. By
+default, this is the meta attribute. If you'd like to construct the
+predicate on the entry's full metadata, then set the "fullmeta" option.
```

Comment:
Is there a way to see the difference between data you'll be using for meta vs fullmeta?

Reply:
I think we should ticket adding that as a new feature of the commands/API.